packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -71,6 +71,7 @@ interface AutocompleteProps {
   onFocus?(): void;
 }
 
+// eslint-disable-next-line max-statements
```

Comment:
This should have a JIRA ticket with the appropriate tech debt tagging on it. Please create one and send it to us in a private Jobber channel. (We don't want to post the JIRA ID here in this repo or in a code comment because this repo is public.)